src/Nethermind/Nethermind.Evm/Tracing/BlockReceiptsTracer.cs

```diff
@@ -78,7 +78,7 @@ protected virtual TxReceipt BuildReceipt(Address recipient, long spentGas, byte
         {
             Logs = logEntries,
             TxType = transaction.Type,
-            Bloom = logEntries.Length == 0 ? Bloom.Empty : new Bloom(logEntries),
+            // Bloom calculated in parallel with other receipts
```

Comment:
Maybe better would be to start a task here, and then await it in BlockProcessor?

Reply:
🤔